git-lp

```diff
@@ -7,6 +7,27 @@
 gitlp_version='0.3'
 
 
+function checksum_bzrlib() {
+local bzrlib_checksum=a4a8558d3c74ca226fbbbbd388d26294
+
+# Perform md5 checksum of bzrlib/index.py to see if has been patched
+local bzrlib_path=`exec python -c "import bzrlib; print bzrlib.__file__"`
+local bzrlib_path=$(dirname "${bzrlib_path}")
+local bzrlib_index="${bzrlib_path}/index.py"
+local checksum=$(md5sum ${bzrlib_index} | cut -f1 -d" ")
+
+if [ "$checksum" != "$bzrlib_checksum" ]
```

Comment:
make that a case statement:
```
case $checksum in
   a4a8558d3c74ca226fbbbbd388d26294)
      ;;
   *)
    bzrlib_bad_checksum=1
esac

if [ $bzrlib_bad_checksum = 1 ]; then
  ...
fi
```

This way we can add more checksums as they are found. All patched versions of bzr are correct, after all.